app/features/feedback/lab/containers/rule-editor-modal-container.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component, PropTypes } from 'react';
+import _ from 'lodash';
+import RuleEditorModal from '../components/rule-editor-modal';
+import defaultValidations from '../helpers/default-validations';
+import strategies from '../../shared/strategies';
+
+class RuleEditorModalContainer extends Component {
+  constructor(props) {
+    super(props);
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSave = this.handleSave.bind(this);
+    this.validateRule = this.validateRule.bind(this);
+
+    this.state = {
+      form: _.assign({}, props.rule),
+      valid: false
+    };
+  }
+
+  handleInputChange(event) {
```

Comment:
I think you can destructure the object in the function parameter directly: `handleInputChange({ target })`

Reply:
👍